### PR TITLE
windows: Fix title bar font for Windows 10

### PR DESCRIPTION
**Pull request by MatinAniss**
This should fix the title bar font for Windows 10 as `Segoe Fluent Icons` is only for Windows 11 and Windows 10 should be using `Segoe MDL2 Assets`, I haven't tested this myself on a Windows 10 machine but the fonts work fine.

Release Notes:

- N/A

### CODE REVIEW

File: crates/ui/src/components/title_bar/windows_window_controls.rs [hunk 1]
```diff
@@ -94,6 +94,25 @@ impl WindowsCaptionButton {
             hover_background_color,
         }
     }
+
+    #[cfg(not(target_os = "windows"))]
+    pub fn get_font() -> &'static str {
```
Comment (maxdeviant) [inline crates/ui/src/components/title_bar/windows_window_controls.rs:99]: These shouldn't need to be `pub`, as we're only using them internally.

File: crates/ui/src/components/title_bar/windows_window_controls.rs [hunk 2]
```diff
@@ -94,6 +94,25 @@ impl WindowsCaptionButton {
...
+    fn get_font() -> &'static str {
+        "Segoe Fluent Icons"
+    }
+
+    #[cfg(target_os = "windows")]
+    fn get_font() -> &'static str {
+        use windows::Wdk::System::SystemServices::RtlGetVersion;
+
+        let mut version = unsafe { std::mem::zeroed() };
+        let status = unsafe { RtlGetVersion(&mut version) };
```
Comment (maxdeviant) [inline crates/ui/src/components/title_bar/windows_window_controls.rs:108]: What's the cost of calling this every render?

Comment (MatinAniss) [reply crates/ui/src/components/title_bar/windows_window_controls.rs]: Unsure on that but made a change to make it better, it seems to run fine.

Comment (maxdeviant) [reply crates/ui/src/components/title_bar/windows_window_controls.rs]: That change you made didn't actually change anything, as the constructor is still called on each render.

We can leave it the way it was, so long as it's not expensive to call potentially on every frame.

**Review by maxdeviant — APPROVED** [review]
Thanks!

**Pull request merged**